src/gt4py/next/program_processors/runners/dace_fieldview/program.py

```diff
@@ -0,0 +1,150 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import dataclasses
+import itertools
+import typing
+from typing import Any, ClassVar, Optional, Sequence
+
+import dace
+import numpy as np
+
+from gt4py._core import definitions as core_defs
+from gt4py.next import allocators, backend as next_backend
+from gt4py.next.ffront import decorator
+from gt4py.next.otf import arguments, toolchain
+from gt4py.next.program_processors.runners.dace_common import utility as dace_utils
+
+
+@dataclasses.dataclass(frozen=True)
+class Program(decorator.Program, dace.frontend.python.common.SDFGConvertible):
+    """Extension of GT4Py Program implementing the SDFGConvertible interface via GTIR."""
+
+    sdfg_closure_cache: dict[str, Any] = dataclasses.field(default_factory=dict)
+    # Being a ClassVar ensures that in an SDFG with multiple nested GT4Py Programs,
+    # there is no name mangling of the connectivity tables used across the nested SDFGs
+    # since they share the same memory address.
+    connectivity_tables_data_descriptors: ClassVar[
+        dict[str, dace.data.Array]
+    ] = {}  # symbolically defined
+
+    def __sdfg__(self, *args: Any, **kwargs: Any) -> dace.sdfg.sdfg.SDFG:
+        if (self.backend is None) or "dace" not in self.backend.name.lower():
+            raise ValueError("The SDFG can be generated only for the DaCe backend.")
+
+        offset_provider = (self.connectivities or {}) | self._implicit_offset_provider
+        column_axis = kwargs.get("column_axis", None)
+
+        gtir_stage = typing.cast(next_backend.Transforms, self.backend.transforms).past_to_itir(
+            toolchain.CompilableProgram(
+                data=self.past_stage,
+                args=arguments.CompileTimeArgs(
+                    args=tuple(p.type for p in self.past_stage.past_node.params),
+                    kwargs={},
+                    column_axis=column_axis,
+                    offset_provider=offset_provider,
+                ),
+            )
+        )
+
+        on_gpu = not allocators.is_field_allocator_for(
+            self.backend.allocator, core_defs.DeviceType.CPU
+        ) and not allocators.is_field_allocator_for(
+            self.backend.allocator, core_defs.DeviceType.CPU_PINNED
+        )
+
+        sdfg = self.backend.executor.step.translation.generate_sdfg(  # type: ignore[attr-defined] # we can assume to get a DaCeTranslationStep here
+            gtir_stage.data,
+            offset_provider=gtir_stage.args.offset_provider,
+            column_axis=kwargs.get("column_axis", gtir_stage.args.column_axis),
```

Comment:
```suggestion
            column_axis=gtir_stage.args.column_axis,
```

I guess at this point we should just use `gtir_stage.args.column_axis` because we passed the keyword argument value when constructing the `gtir_stage`, see line 49..